app/controllers/topics_controller.rb

```diff
@@ -1314,16 +1314,21 @@ def track_visit_to_topic
       TopicsController.defer_add_incoming_link(hash)
     end
 
-    TopicsController.defer_track_visit(topic_id, ip, user_id, track_visit)
+    TopicsController.defer_track_visit(topic_id, user_id, track_visit)
   end
 
-  def self.defer_track_visit(topic_id, ip, user_id, track_visit)
+  def self.defer_track_visit(topic_id, user_id, track_visit)
     Scheduler::Defer.later "Track Visit" do
```

Comment:
Nitpicking / micro-optimizing, I wonder if we should bypass the whole "defer.later" block if `track_visit` is false? 🤔 

eg.

```ruby
def self.defer_track_visit(topic_id, user_id, track_visit)
  return unless track_visit

  Scheduler::Defer.later "Track Visit" do
    TopicUser.track_visit!(topic_id, user_id)
  end
end
```

Reply:
Or even: should we even bother calling the `defer_track_visit` method if we aren't going to track the visit anyways? 🧐

Reply:
Oh yeah you're right! I missed that in the refactor, this can be removed now